FileAPI/url/cross-global-revoke.sub.html

```diff
@@ -0,0 +1,63 @@
+<!doctype html>
+<meta charset="utf-8">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<body>
+<script>
+async_test(t => {
+  const blob_contents = 'test blob contents';
+  const blob = new Blob([blob_contents]);
+  const url = URL.createObjectURL(blob);
+  const frame = document.createElement('iframe');
+  frame.setAttribute('style', 'display:none;');
+
+  self.addEventListener('message', t.step_func(e => {
+    if (e.source != frame.contentWindow) return;
```

Comment:
nit: would prefer `!==` to `!=` everywhere, so the reader doesn't have to reason about type coercion.

Reply:
Does that actually work with event.source and frame.contentWindow ? I thought it was not recommended to use === to compare proxy objects.

Reply:
Are you asking whether `===` works as opposed to `==`, or whether the equality operator works at all?

Reply:
The former. I know there are some special hacks to make == work, but couldn't remember whether === is guaranteed to consider the objects equal too.

Reply:
Thanks! Looks like this all got clarified last year or so https://blog.whatwg.org/windowproxy-window-and-location and I missed it. Sorry for the noise!